var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -29,7 +29,6 @@ class Mfem(Package):
     """Free, lightweight, scalable C++ library for finite element methods."""
 
     homepage = 'http://www.mfem.org'
-    url      = 'https://github.com/mfem/mfem'
```

Comment:
Use of `goo.gl` URL shortening service in Spack URLs is not appropriate.  Can this be removed and reverted to the full URL?